lametro/views.py

```diff
@@ -689,8 +693,31 @@ def get_context_data(self, **kwargs):
         else:
             context['sponsored_legislation'] = []
 
-        committees_lst = [action._organization.name for action in person.committee_sponsorships]
-        context['committees_lst'] = list(set(committees_lst))
+        with connection.cursor() as cursor:
+
+            sql = ('''
+                SELECT o.name as organization, o.slug as org_slug, m.role
+                FROM councilmatic_core_membership AS m
+                JOIN councilmatic_core_organization AS o
+                ON o.ocd_id = m.organization_id
+                WHERE m.person_id = %s
+                AND m.end_date::date > NOW()::date
+                AND m.organization_id != %s
+                ORDER BY
+                    CASE
+                        WHEN m.role='Chair' THEN 0
+                        WHEN m.role='Vice Chair' THEN 1
+                        WHEN m.role='Member' THEN 2
```

Comment:
The role is never null I take it? I seem to remember PostgreSQL putting NULLs first by default.

Reply:
okay - that's a good question. Looks like the role cannot be Null, since the scraper guarantees that (i.e., assigning a value of "Member" of "Non-voting member"): https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/people.py#L117

So, this query should work as expected.